doc/rst/technotes/libraries.rst

```diff
@@ -269,6 +269,14 @@ requires the following:
 - ``Cython``
 - ``numpy``
 
+If you are on a system where libraries are built statically by default (e.g.
+not OSx), you will need to set the environment variable ``CHPL_LIB_PIC`` to
+``pic`` and rebuild your Chapel compiler.  This will cause the Chapel runtime
+and third-party libraries to be built with position independent code, which
+Python requires.  Note that position independent code will likely encounter some
+performance degradation as opposed to normal Chapel code.  For this reason, we
+recommend only using ``CHPL_LIB_PIC=pic`` when interoperating with Python.
```

Comment:
` when interoperating with Python` => `when position independent code is required (e.g. when calling Chapel from Python)`? (since there may be other non-python use-cases)